content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -143,14 +140,14 @@ $ istioctl analyze -k --all-namespaces --suppress "IST0102=Namespace frod" --sup
 
 You can also ignore specific analyzer messages using an annotation on the resource. For example, to ignore code IST0107 (`MisplacedAnnotation`) on resource `deployment/my-deployment`:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=annotate_for_deployment_suppression >}}
 $ kubectl annotate deployment my-deployment galley.istio.io/analyze-suppress=IST0107
 {{< /text >}}
 
 To ignore multiple codes for a resource, separate each code with a comma:
 
-{{< text bash >}}
-$ kubectl annotate deployment my-deployment galley.istio.io/analyze-suppress=IST0107,IST0002
+{{< text syntax=bash snip_id=annotate_for_deployment_suppression_107 >}}
+$ kubectl annotate deployment my-deployment galley.istio.io/analyze-suppress=IST0107,IST0002 --overwrite
```

Comment:
Instead of adding `--overwrite` in the doc, you should just remove the IST0107 suppress in the test.sh before calling this snip function. Since the doc describes this snip and the previous one as either or (as opposed to step by step run this, then run that), the reader might be confused why `--overwrite` is needed in the second snip but not the first. In general, you should try not to change the commands in the doc unless your fixing a bug or it makes something more clear for the reader.
```suggestion
$ kubectl annotate deployment my-deployment galley.istio.io/analyze-suppress=IST0107,IST0002
```